tkg/client/management_components.go

```diff
@@ -120,7 +141,11 @@ func (c *TkgClient) getTKGPackageConfigValuesFile(managementPackageVersion, kube
 		return "", err
 	}
 
-	valuesFile, err := managementcomponents.GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion, userProviderConfigValues, tkgBomConfig)
+	addonsManagerPackageVersion, err := c.getAddonsManagerPackageversion(managementPackageVersion)
+	if err != nil {
+		return "", err
```

Comment:
should we treat this as an error and return OR should we just catch the error and leave manageentPackageVersion as is?

Reply:
Treating error sounds good to me. We dont upgrade to finish with wrong version of addons-manager